code_examples/sdk_examples/src/core_features/did/08_full_did_tx.ts

```diff
@@ -0,0 +1,19 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+export async function signDidExtrinsic(
```

Comment:
This functio not only signs but also submits, so it should be renamed.

Reply:
What about having these functions return the extrinsic object?

Reply:
I would at least sign them using an account. This makes it clear that it's a two step process.

Reply:
[55eefba](https://github.com/KILTprotocol/docs/pull/183/commits/55eefbae2f0a781b498b72e71481d55342609509)